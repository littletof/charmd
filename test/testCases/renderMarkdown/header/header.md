# h1 Heading

h1 Heading with `===`
===

## h2 Heading

h2 Heading with `---`
---

### h3 Heading
#### h4 Heading
##### h5 Heading
###### h6 Heading
