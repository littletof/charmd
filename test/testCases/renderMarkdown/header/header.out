[1m[4m[31m# h1 Heading[39m[24m[22m

[1m[4m[31m# h1 Heading with [7m === [27m[39m[24m[22m

[33m[1m## h2 Heading[22m[39m

[33m[1m## h2 Heading with [7m --- [27m[22m[39m

[32m[1m### h3 Heading[22m[39m

[35m[1m#### h4 Heading[22m[39m

[36m[1m##### h5 Heading[22m[39m

[34m[1m###### h6 Heading[22m[39m
